omod-1.12/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/controller/openmrs1_12/OrderGroupController1_12Test.java

```diff
@@ -0,0 +1,199 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.module.webservices.rest.web.v1_0.controller.openmrs1_12;
+
+
+import java.util.ArrayList;
+import java.util.List;
+
+
+
+import org.apache.commons.beanutils.PropertyUtils;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.openmrs.Encounter;
+import org.openmrs.Order;
+import org.openmrs.OrderGroup;
+import org.openmrs.OrderSet;
+import org.openmrs.Patient;
+import org.openmrs.api.OrderService;
+import org.openmrs.api.context.Context;
+import org.openmrs.module.webservices.rest.SimpleObject;
+import org.openmrs.module.webservices.rest.web.RestTestConstants1_12;
+import org.openmrs.module.webservices.rest.web.v1_0.controller.MainResourceControllerTest;
+import org.springframework.mock.web.MockHttpServletRequest;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.openmrs.module.webservices.rest.test.Util;
+
+public class OrderGroupController1_12Test  extends MainResourceControllerTest {
+  
+	private OrderService orderService;
+	
+	String patient_uuid =  RestTestConstants1_12.ORDER_GROUP_PATIENT_UUID;
+	
+	String encounter_uuid = RestTestConstants1_12.ORDER_GROUP_ENCOUNTER_UUID;
+	
+	String orderSet_uuid = RestTestConstants1_12.ORDER_GROUP_ORDERSET_UUID;
+	
+	String display = RestTestConstants1_12.ORDER_GROUP_DISPLAY;
+	
+	String orderGroup_uuid = RestTestConstants1_12.ORDER_GROUP_UUID;
+	
+	String json = "{\n" + 
```

Comment:
Do all these need to be class level variables?

Reply:
i updated my Pr

Reply:
i have removed the uncesary class level variables